docs/source/resources/tutorial-template.rst

```diff
@@ -0,0 +1,81 @@
+=================
+Tutorial template
+=================
+
+.. contents::
+   :local:
+   :depth: 1
+   
+.. _documentation@anaconda.com: documentation@anaconda.com
+
+*This document describes the steps for creating*
+*a tutorial for conda build*.
+
+* *Follow the* :doc:`writing style guide <style-guide>`.
+* *Copy the template*: https://github.com/conda/conda-build/tree/master/docs/source/resources/tutorial-template.rst.
+* *Replace the italicized text with your tutorial content*.
+* *Review other* :doc:`tutorials <../user-guide/tutorials/index>`
+  *for additional guidance*.
+* *Contact us at* `documentation@anaconda.com`_ *for help*.
+
+Overview
+========
+
+*Provide descriptions of the tutorial's*:
+    * *Purpose*.
+    * *Benefits*.
+    * *Application*.
+
+
+Who is this for?
+================
+
+* *Who is your audience*?
+* *What skills or prior knowledge do they need*?
+* *How will they use this tutorial*?
+
+Before you start
+================
+
+Before you start, check the :ref:`prereqs`.
+
+
+*Provide descriptions of and links to requisite programs*.
+
+Glossary (optional)
+-------------------
+
+*If you're using several technical terms that your*
+*readers may be unfamiliar with, provide a glossary*
+*of key terms*.
+
+
+.. list-table::
+    :widths: 10 50
+    :header-rows: 1
+    :stub-columns: 1
+
+    * - Glossary
+      - Definition
+    * - Term 1
+      - Term 1 defined
+    * - Term 2
+      - Term 2 defined
+    * - Term 3
+      - Term 3 defined
+
+Tutorial tasks
+==============
+
+* *Provide descriptive titles for each step*.
```

Comment:
Do you mean for each "section" or "tasks"? I think of a step as one thing they do (like you use it below in "Write a series of steps...")